fipy/matrices/pysparseMatrix.py

```diff
@@ -246,6 +272,126 @@ def exportMmf(self, filename):
         """
         self.matrix.export_mtx(filename)
 
+    @property
```

Comment:
Might want to pylint some of this or run black on it.

Reply:
I thought I was linting, but it turns out pylint and black only run on setup.py. I guess I stubbed those when I first set up the CIs and then never finished the job. Too big to take on now (#604), but I did lint these and clean up some stuff. Interestingly, none of the trailing whitespace was introduced in this pull request. A bunch of it comes from the way matrices are formatted as strings, but I cleaned up the rest.